### ISSUE TITLE
Install udev rules

### ISSUE THREAD

**Issue by Tobias-Fischer** [conda-forge/libcaer-feedstock#1]
Installation is currently disabled

@wolfv - any ideas how to deal with that? If I remember correctly you had some thoughts on how to do that in conda-forge.

**Comment by wolfv** [comment]
Hmm... the only way would be via post-link script right now. You can execute arbitrary commands with a post link script (and post-unlink script). This script could create a /etc/udev/rules.d/10-conda-libcaer.rules file

**Comment by Tobias-Fischer** [comment]
But don't you need root permissions for creating these files? How would that work?

**Comment by wolfv** [comment]
Right, you'd need root permissions. It's probably possible to make the post link script ask the User if the installation is wanted.

Otherwise we could add a small helper script in $PREFIX/bin/install_udev.sh and explain users that they'll have to run that after installation

**Comment by wolfv** [comment]
Or we make a conda-wide plugin where udev rules added in $PREFIX/udev/* are automatically installed & removed -- but that would take work in mamba or conda to achieve (conda is looking to add plugin support and we'd like to do the same in mamba)
